lib/telepoison.ex

```diff
@@ -318,4 +318,8 @@ defmodule Telepoison do
       end
     )
   end
+
+  def strip_uri_credentials(uri) do
+    uri |> URI.parse() |> Map.put(:userinfo, nil) |> Map.put(:authority, nil) |> URI.to_string()
```

Comment:
Isn't better to put something like a placeholder for userinfo and authority fields?